acvm-repo/acir/src/circuit/opcodes.rs

```diff
@@ -30,7 +31,8 @@ impl BlockType {
 
 #[allow(clippy::large_enum_variant)]
 #[derive(Clone, PartialEq, Eq, Serialize, Deserialize, Hash)]
-pub enum Opcode<F> {
+#[cfg_attr(feature = "arb", derive(proptest_derive::Arbitrary))]
+pub enum Opcode<F: AcirField> {
```

Comment:
:grimacing: Is this trait bound really necessary?

Reply:
:face_exhaling: seems like a really leaky solution to just needing a `Debug` implementation for `Arbitrary`

Reply:
Yes, it's annoying. I thought I'm not adding any bound that won't be ultimately satisfied, so it's just making what would inevitably be true official, but it's certainly viral.